pkg/api/pod/warnings.go

```diff
@@ -234,21 +234,7 @@ func warningsForPodSpecAndMeta(fieldPath *field.Path, podSpec *api.PodSpec, meta
 		warnings = append(warnings, fmt.Sprintf("%s: fractional byte value %q is invalid, must be an integer", fieldPath.Child("spec", "overhead").Key(string(api.ResourceEphemeralStorage)), value.String()))
 	}
 
-	// use of pod seccomp annotation without accompanying field
```

Comment:
we should keep emitting these warnings to make it clear they're doing something that isn't working as they expect